_includes/header.html

```diff
@@ -38,6 +38,7 @@
                 </ul>
             </li>
             <li><a href="/team">The Team</a></li>
+            <li><a href="/resources">The Team</a></li>
```

Comment:
These both say "The Team." Did you test this first?

Also, this should be under More.